go.mod

```diff
@@ -1,18 +1,14 @@
 module github.com/kyma-project/helm-broker
 
-go 1.16
+go 1.17
```

Comment:
I just realized the binaries are built natively and then copied to the images. We may need to bump the CI/CD pipelines version of go